packages/namespaces-react/src/withNamespaceProvider.js

```diff
@@ -0,0 +1,19 @@
+import { mergeLeft } from 'ramda';
+import React from 'react';
+
+import NamespaceProvider from './NamespaceProvider';
+
+const withNamespaceProvider = optionProps => WrappedComponent => ({
+	feature,
+	namespace,
+	store,
+	useNamespace,
+	children,
+	...otherProps
+}) => (
+	<NamespaceProvider {...mergeLeft(optionProps, { feature, namespace, store, useNamespace })}>
```

Comment:
i think we should pass all props down to `WrappedComponent`, so i think we should use `mergeLeft(options, props)` and just pass props down (no need to destructure any of them).

add a comment above <NamespaceProvider {...} usage
```
// NOTE: NamespaceProvider will ignore any unknown props.